Dockerfile-debug

```diff
@@ -0,0 +1,8 @@
+FROM gcr.io/linkerd-io/base:2019-02-19.01
+RUN apt-get update && apt-get install -y --no-install-recommends \
+    tcpdump \
+    iproute2 \
+    lsof \
+    tshark && \
+    rm -rf /var/lib/apt/lists/*
+ENTRYPOINT [ "tshark", "-i", "any" ]
```

Comment:
is the intent that this starts tshark immediately?

Reply:
That's correct. I can definitely see a feature where we can provide arguments to the container when we inject so that we aren't listening to everything for the particular pod. But I think this might be a good start.